src/Microsoft.AspNet.Mvc.ModelBinding/Binders/CompositeModelBinder.cs

```diff
@@ -129,12 +132,6 @@ private static bool IsBindingAtRootOfObjectGraph(ModelBindingContext bindingCont
                 PropertyFilter = oldBindingContext.PropertyFilter,
             };
 
-            // validation is expensive to create, so copy it over if we can
```

Comment:
Do we know when reusing the validation node might have made sense in the past? Is this a perf \ security issue we are inadvertently creating here?